seed/challenges/01-front-end-development-certification/bootstrap.json

```diff
@@ -572,8 +572,8 @@
         "</div>"
       ],
       "tests": [
-        "assert($(\"button\").hasClass(\"btn-primary\"), 'message: Your button should have the class <code>btn-primary</code>.');",
-        "assert($(\"button\").hasClass(\"btn-block\") && $(\"button\").hasClass(\"btn\"), 'message: Your button should still have the <code>btn</code> and <code>btn-block</code> classes.');",
+        "assert($(\"button\").first().hasClass(\"btn-primary\"), 'message: Your <code>Like</code> button should have the class <code>btn-primary</code>.');",
+        "assert($(\"button\").first().hasClass(\"btn-block\") && $(\"button\").hasClass(\"btn\"), 'message: Your <code>Like</code> button should still have the <code>btn</code> and <code>btn-block</code> classes.');",
```

Comment:
This line and the previous, you've added a `.first()` method. This line, 576, essentially has two checks. Did you also mean to add the `.first()` method to the `$("button").hasClass("btn")` check as well?